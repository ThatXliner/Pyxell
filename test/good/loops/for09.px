
for x in 1.75..3.75 do
    print x
