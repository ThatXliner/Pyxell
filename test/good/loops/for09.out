1.75
2.75
3.75
